components/modelregistry/modelregistry.go

```diff
@@ -79,7 +86,10 @@ func (m *ModelRegistry) ReconcileComponent(ctx context.Context, cli client.Clien
 
 		// Update image parameters only when we do not have customized manifests set
 		if (dscispec.DevFlags == nil || dscispec.DevFlags.ManifestsUri == "") && (m.DevFlags == nil || len(m.DevFlags.Manifests) == 0) {
-			if err := deploy.ApplyParams(Path, imageParamMap, false); err != nil {
+			extraParamsMap := map[string]string{
```

Comment:
Yes, this overwrites the default empty string, so that the MR operator can use the secret name from odh operator.

Reply:
It's just a string that is passed along. I've tested it locally just for the MR operator by setting the value in my local params.env, if that's what you are asking.

Reply:
now i got confused, 
if all in all, you just need static
```
if err := deploy.ApplyParams(Path, imageParamMap, map[string]string{"DEFAULT_CERT":"default-modelregistry-cert"}); err != nil {
				return fmt.Errorf("failed to update image from %s : %w", Path, err)
}
```
?

or you could simply set in your params.env dirctly with:
`DEFAULT_CERT=default-modelregistry-cert`
then keep 
```
if err := deploy.ApplyParams(Path, imageParamMap); err != nil {
				return fmt.Errorf("failed to update image from %s : %w", Path, err)
}
```

Reply:
It's for the future, when we realize that we can just create a single default cert across components, instead of having a cert per component. By having a configurable property, we won't want to have to change the MR operator to do that just the odh operator. :smile:

Reply:
@dhirajsb  is it for non-DevFlags case only (like images) or with DevFlags as well, like extra params for other components?